### PR TITLE
doc: add testnet4 section header for config file

### PR DESCRIPTION
**Pull request by MarnixCroes**
This PR adds the missing `testnet4` config section header documentation for the config file.

### CODE REVIEW

**Comment by Sjors** [comment]
utACK 605926da0ab4ac7ae4e9aed55a059f37c31c15b5

File: doc/bitcoin-conf.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Comments may appear in two ways:
 ### Network specific options
 
 Network specific options can be:
-- placed into sections with headers `[main]` (not `[mainnet]`), `[test]` (not `[testnet]`), `[signet]` or `[regtest]`;
+- placed into sections with headers `[main]` (not `[mainnet]`), `[test]` (not `[testnet]`, for testnet3), `testnet4`, `[signet]` or `[regtest]`;
```
[the remark this answers is not in the document]
Comment (MarnixCroes) [reply doc/bitcoin-conf.md]: fixed, my bad...

**Comment by Sjors** [comment]
ACK 61cdb1c9d83778b95f4f9596f34617b7a191d0a5

**Review by pablomartin4btc — COMMENTED** [review]
ACK 61cdb1c9d83778b95f4f9596f34617b7a191d0a5

`BUILDDIR` would need to be updated in `contrib/devtools/gen-bitcoin-conf.sh`, otherwise `bitcoind` won't be found.

**Comment by fjahr** [comment]
ACK 61cdb1c9d83778b95f4f9596f34617b7a191d0a5

**Review by tdb3 — APPROVED** [review]
ACK 61cdb1c9d83778b95f4f9596f34617b7a191d0a5

Good find.  Thanks.

**Pull request merged**